test/integration/index.spec.js

```diff
@@ -2125,77 +2125,120 @@ describe("Test CORS", () => {
 	});
 
-	it("errors on mismatching origin header", () => {
+	// breaking change /!\ default is super secure now. no cors means... no cors.
+	it("with default settings", () => {
 		[broker, service, server] = setup({
-			cors: {
-				origin: "a"
-			}
+			cors: {},
 		});
-		return broker.start()
-			.then(() => request(server)
-				.get("/test/hello")
-				.set("Origin", "http://localhost:3000"))
-			.then(res => {
+
+		return broker
+			.start()
+			.then(() =>
+				request(server)
+					.get("/test/hello")
+					.set("Origin", "http://localhost:3000")
+			)
+			.then((res) => {
 				expect(res.statusCode).toBe(403);
 				expect(res.body).toEqual({
-					"message": "Forbidden",
-					"code": 403,
-					"type": "ORIGIN_NOT_ALLOWED",
-					"name": "ForbiddenError"
+					message: "Forbidden",
+					code: 403,
+					type: "ORIGIN_NOT_ALLOWED",
+					name: "ForbiddenError",
 				});
-			}).then(() => broker.stop()).catch(err => broker.stop().then(() => { throw err; }));
+			})
+			.then(() => broker.stop())
+			.catch((err) =>
+				broker.stop().then(() => {
+					throw err;
+				})
+			);
 	});
 
-	it("with default settings", () => {
+	it("with custom global settings (boolean)", () => {
 		[broker, service, server] = setup({
-			cors: {}
+			cors: {
+				origin: true,
+			},
 		});
 
-		return broker.start()
-			.then(() => request(server)
-				.get("/test/hello")
-				.set("Origin", "http://localhost:3000"))
-			.then(res => {
+		return broker
+			.start()
+			.then(() =>
+				request(server)
+					.get("/test/hello")
+					.set("Origin", "http://localhost:3000")
+			)
+			.then((res) => {
 				expect(res.statusCode).toBe(200);
-				expect(res.headers["content-type"]).toBe("application/json; charset=utf-8");
 				expect(res.headers["access-control-allow-origin"]).toBe("*");
-
-				expect(res.body).toBe("Hello Moleculer");
 			})
-			.then(() => broker.stop()).catch(err => broker.stop().then(() => { throw err; }));
+			.then(() => broker.stop())
+			.catch((err) =>
+				broker.stop().then(() => {
+					throw err;
+				})
+			);
 	});
 
 	it("with custom global settings (string)", () => {
 		[broker, service, server] = setup({
 			cors: {
 				origin: "http://localhost:3000",
-				exposedHeaders: "X-Response-Time",
-				credentials: true
-			}
+			},
 		});
 
-		return broker.start()
-			.then(() => request(server)
-				.get("/test/hello")
-				.set("Origin", "http://localhost:3000"))
-			.then(res => {
+		return broker
+			.start()
+			.then(() =>
+				request(server)
+					.get("/test/hello")
+					.set("Origin", "http://localhost:3000")
+			)
+			.then((res) => {
 				expect(res.statusCode).toBe(200);
-				expect(res.headers["content-type"]).toBe("application/json; charset=utf-8");
 				expect(res.headers["access-control-allow-origin"]).toBe("http://localhost:3000");
-				expect(res.headers["access-control-allow-credentials"]).toBe("true");
-				expect(res.headers["access-control-expose-headers"]).toBe("X-Response-Time");
```

Comment:
Why do you remove checking other header properties?

Reply:
I took to remove them because I think they're not what the test is testing for. In my opinion the test cases for this feature (cors policy) should be for checking that "given a cors settings and an http request as input, the proper "allow-origin" header is returned." ; so then those headers are not necessary to be included in the test.

Please let me know if you want them back :)

Reply:
Yeah, but testing the `access-control-allow-credentials` and `access-control-expose-headers` headers is only here. So if you remove, you should add other tests for these header values.